data-imports/clean_utils.py

```diff
@@ -139,7 +139,9 @@ def clean_bbl(table, boro, block, lot):
     return sql + '''
     UPDATE {table} SET {block} = REPLACE({block}, ' ', 0);
     UPDATE {table} SET {block} = REPLACE({block}, '`', '');
+    UPDATE {table} SET {block} = preg_replace('/[^0-9]/', '', {block});
```

Comment:
Great! However, I'd remove the 2 lines above, since the regex takes care of them both!